### ISSUE TITLE
what is the purpose of this plugin ?

### ISSUE THREAD

**Issue by prazdevs** [pinia-orm/pinia-plugin-persistence#3]
Hi :) I'm the author of [this plugin](https://github.com/prazdevs/pinia-plugin-persistedstate) and writer of ~70% of its codebase.

I wanted to understand why most of the code is duplicated here with no credit given to anyone (me and contributors). The project not even being a fork.

Did you encounter issues with the original ? Maybe i could be of any help, let me know.

(Btw there are still some typos in the docs, and you forgot to change the name in the nuxt module metadata... 🤔 )

**Comment by elonehoo** [comment]
Yes, I think I forgot to add this. This is my code that imitated yours a little bit in the process of learning, sorry

**Comment by elonehoo** [comment]
Sorry

**Comment by elonehoo** [comment]
It's really my fault, I'm really sorry
